pilot/pkg/networking/core/v1alpha3/envoyfilter/cluster_patch_test.go

```diff
@@ -91,6 +93,23 @@ func Test_clusterMatch(t *testing.T) {
 			},
 			want: false,
 		},
+		{
+			name: "service name match for inbound cluster",
+			args: args{
+				proxy:     &model.Proxy{Type: model.SidecarProxy},
+				operation: networking.EnvoyFilter_Patch_MERGE,
+				matchCondition: &networking.EnvoyFilter_EnvoyConfigObjectMatch{
+					ObjectTypes: &networking.EnvoyFilter_EnvoyConfigObjectMatch_Cluster{
+						Cluster: &networking.EnvoyFilter_ClusterMatch{
+							Service: "foo.bar",
+						},
+					},
+				},
+				cluster: &cluster.Cluster{Name: "inbond|80||"},
```

Comment:
nit (or maybe not a nit, seems like it could impact the test): inbond -> inbound

Reply:
Great catch. Fixed. PTAL